app/controllers/AmendsController.scala

```diff
@@ -50,327 +51,357 @@ object AmendsController extends AmendsController {
                 val updatedTotal = updated.copy(relevantAmount = Some(Helpers.totalValue(updated)))
                 val amendProtModel = AmendProtectionModel(model.originalProtection, updatedTotal)
 
-                keyStoreConnector.saveFormData[AmendProtectionModel](Strings.keyStoreProtectionName(updated), amendProtModel).map{
+                keyStoreConnector.saveFormData[AmendProtectionModel](Strings.keyStoreProtectionName(updated), amendProtModel).map {
                   _ => Redirect(routes.AmendsController.amendsSummary(updated.protectionType.get.toLowerCase, updated.status.get.toLowerCase))
                 }
 
               case _ =>
-                Logger.warn(s"Could not retrieve amend protection model for user with nino ${user.nino} after submitting amend pensions taken before")
+                Logger.warn(s"Could not retrieve amend protection model for user with nino $nino after submiiting amend pensions takne between")
                 Future.successful(InternalServerError(views.html.pages.fallback.technicalError(ApplicationType.existingProtections.toString)).withHeaders(CACHE_CONTROL -> "no-cache"))
             }
           }
         }
       )
-  }
 
-  def amendPensionsTakenBetween(protectionType: String, status: String): Action[AnyContent] = AuthorisedByAny.async { implicit user => implicit request =>
-    amendRoute(AmendJourney.pensionTakenBetween, protectionType, status).apply(request)
+    }
   }
 
-  val submitAmendPensionsTakenBetween = AuthorisedByAny.async { implicit user => implicit request =>
-    amendPensionsTakenBetweenForm.bindFromRequest.fold(
-      errors => {
-        val form = errors.copy(errors = errors.errors.map { er => FormError(er.key, Messages(er.message)) })
-        Future.successful(BadRequest(pages.amends.amendPensionsTakenBetween(form)))
-      },
-      success => {
-        val validatedForm = AmendPensionsTakenBetweenForm.validateForm(amendPensionsTakenBetweenForm.fill(success))
-        if (validatedForm.hasErrors) {
-          Future.successful(BadRequest(pages.amends.amendPensionsTakenBetween(validatedForm)))
-        } else {
-          keyStoreConnector.fetchAndGetFormData[AmendProtectionModel](Strings.keyStoreAmendFetchString(success.protectionType, success.status)).flatMap {
-            case Some(model) =>
-              val updatedAmount = success.amendedPensionsTakenBetween match {
-                case "yes" => success.amendedPensionsTakenBetweenAmt.get.toDouble
-                case "no" => 0.asInstanceOf[Double]
-              }
-              val updated = model.updatedProtection.copy(postADayBenefitCrystallisationEvents = Some(updatedAmount))
-              val updatedTotal = updated.copy(relevantAmount = Some(Helpers.totalValue(updated)))
-              val amendProtModel = AmendProtectionModel(model.originalProtection, updatedTotal)
+  def amendOverseasPensions(protectionType: String, status: String): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
Can we make amendRoute un-authed and add auth to all actions calling amendRoute?